cdc/kv/region_worker.go

```diff
@@ -390,6 +446,17 @@ func (w *regionWorker) handleEventEntry(
 			}
 			metricPullEventInitializedCounter.Inc()
 			state.initialized = true
+			select {
+			case w.rtsUpdateCh <- &regionResolvedTs{regionID: regionID, resolvedTs: state.sri.ts}:
+			default:
+				// rtsUpdateCh block often means too many regions are suffering
+				// lock resolve, the kv client status is not very healthy.
+				log.Warn("region is not upsert into rts manager", zap.Uint64("region-id", regionID))
```

Comment:
What is "upsert"?

Reply:
It is a special syntax in some RDBMS, which means `insert or update`, ref: https://en.wikipedia.org/wiki/Merge_(SQL)